### PR TITLE
Add previous and next day functionality to diary

### PR DESCRIPTION
**Pull request by TheKaterTot**
This adds the Previous and Next buttons to the diary, which allow you to navigate through dates and see the meals for that day. On refresh, the page returns to the current date, as per the user story. All other data persists.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <body>
   <div class="container">
     <h1 class="center">Diary</h1>
-    <h3 id="date" class="center"></h3>
+    <h3 class="center"><a href="#" class="button button-clear" id="previous-day">Previous</a> <span id="date"></span> <a href="#" class="button button-clear" id="next-day">Next</a></h3>
```
Comment (cmacaulay) [inline index.html:13]: 🎉 a button! ⏭

File: lib/index.js [hunk 2]
```diff
@@ -1,14 +1,13 @@
 const moment = require('moment');
 const $ = require('jquery');
-const currentDate = moment();
+let currentDate = moment();
```
Comment (cmacaulay) [inline lib/index.js:3]: 🎉

Comment (cmacaulay) [reply lib/index.js:3]: yay for dynamism!

File: lib/index.js [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
 
 $(document).ready(function() {
   setDate();
-  fetchBreakfast(currentDate);
-  fetchLunch(currentDate);
-  fetchDinner(currentDate);
-  fetchSnacks(currentDate);
+  fetchDiaryInfo();
+  $('#previous-day').on('click', previousDay)
+  $('#next-day').on('click', nextDay)
```
Comment (cmacaulay) [inline lib/index.js:10]: 🗡 🎯 listenin' to those events

File: lib/index.js [hunk 4]
```diff
@@ -51,3 +50,23 @@ function fetchDinner(date) {
...
+function fetchDiaryInfo() {
+  $('tbody.foods').html('');
+  fetchBreakfast(currentDate);
+  fetchLunch(currentDate);
+  fetchDinner(currentDate);
+  fetchSnacks(currentDate);
+}
+
+function previousDay() {
+  currentDate = currentDate.subtract(1, "days");
```
Comment (cmacaulay) [inline lib/index.js:63]: @TheKaterTot is this subtract built in? Do you mind explaining?

File: lib/index.js [hunk 5]
```diff
@@ -51,3 +50,23 @@ function fetchDinner(date) {
...
+  setDate();
+  fetchDiaryInfo();
+}
+
+function nextDay() {
+  currentDate = currentDate.add(1, "days");
+  setDate();
+  fetchDiaryInfo();
```
Comment (cmacaulay) [inline lib/index.js:71]: 🎉

**Review by cmacaulay — APPROVED** [review]
Code is really easy to follow, all of it makes sense! Thank you so much for all of your hard work on this.

File: lib/index.js (hunk 4 above)
Comment (TheKaterTot) [reply lib/index.js:63]: This is a method available from momentjs to do calculations on dates. It makes my heart go 💥

**Pull request merged**